packages/web/components/UserProfile/PageLink.js

```diff
@@ -15,13 +15,14 @@ const PageLink = ({ active, href, children, onClick }) => (
 
 PageLink.propTypes = {
 	href: PropTypes.string.isRequired,
-	active: PropTypes.bool.isRequired,
+	active: PropTypes.bool,
```

Comment:
This doesn't needs to be mandatory as user profile dropdown use this component too without an active link